pkgs/desktops/gnome/extensions/customize-ibus/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub, gnome, gettext, glib, ibus }:
```

Comment:
You say that the package depends on `ibus` and I see you adding the build input here, but that variable is actually never used in the package. If you say that the package works for you and that it works, could it be that `ibus` is a pure run time dependency and you simply need to have it in PATH?

Reply:
Yes, `ibus` is a pure run time dependency and it doesn't need to be used when building.

Reply:
If the dependency is required for the extension to work at all, we usually patch the source paths to hard-code against `${ibus}/bin/ibus` or something.

Also, if this is the only issue with this extension, you may be able to get things to work by overriding the automatically packaged extension in `extensionOverride.nix` instead of doing a full manual package.

Reply:
Thanks for your suggestions!

> If the dependency is required for the extension to work at all, we usually patch the source paths to hard-code against ${ibus}/bin/ibus or something.

Actually the extension uses some features in GNOME https://github.com/openSUSE/Customize-IBus/blob/aae8df8281834fd4aa641e725f99a1aa47c9d50a/customize-ibus%40hollowman.ml/extension.js#L30-L35 that need IBus to be available https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/misc/ibusManager.js . Make source patches only to the extension itself won‘t help. I guess the only way to solve this issue is to explicitly specify the dependency.

Reply:
> Make source patches only to the extension itself won‘t help. I guess the only way to solve this issue is to explicitly specify the dependency.

I don't understand. Simply adding the variable to the list at the top does nothing, and so it won't help. You need to somehow use it to have an effect (Nix is lazy). On the other hand, we have little options other thane patching the source.

Reply:
> You need to somehow use it to have an effect (Nix is lazy). 
OK, I see.
> On the other hand, we have little options other thane patching the source.
Will look into it later, or just use a workaround to use IBus in the source code :-)

Reply:
> We do not need to do anything about the GNOME Shell dependencies as extensions run in Shell context.

OK, I see. 

> Though notify-send should probably be replaced by GNotification or even some more reasonable UI.

I will directly commit to the upstream and draft a new release for that case.

So close this PR now since I guess issues mentioned has been resolved.